static/docs/user-guide/external-data/index.md

```diff
@@ -0,0 +1,22 @@
+# External Data
+
+External data are located outside the project directory, either on the local
```

Comment:
I think we had better explanation before why and when it should be used.

Reply:
Do you mean this explanation:

> There are cases when data is so large, or its processing is organized in a way that you would like to avoid moving it out of its external/remote location. For example from a network attached storage (NAS) drive, processing data on HDFS, running Dask via SSH, or having a script that streams data from S3 to process it. A mechanism for external outputs and external dependencies provides a way for DVC to control data externally.

Reply:
Yes.